db.json

```diff
@@ -1,12 +1,212 @@
 {
     "menus": {
         "title_screen": "_____________________________________________________  \n|  #########################            _.._  __     | \n|   Welcome to the Text RPG         .--{    }/  \"~-._| \n|  #########################       /    ^++^/      / | \n|        ${o3} - Play -               {        (0     /  | \n|                                  \\=.___ .//\\___+\"  | \n|                                  \\   .//'     /    | \n|        ${o2} - Help -                )  _//'    _(     | \n|                                  (HHHHH[]HHHH)     | \n|                                  / \\   ..  / \\     | \n|        ${o1} - Quit -              /   }  .. {    \\    | \n|                                 ^+._/\\ .. /\\_.+^   | \n|        copyright 2020                 \\__/         | \n|____________________________________________________| \n",
-        "help": "############################               \n#       -HELP -          #\n############################               \n                                           \nUse up down left right to move\n           \nType commands to use them\n                \nUse \"look\" to inspect\n                    \nAnd most importantly be careful out there\n\npress \"q\" to go back ---->\n",
+        "help": "############################               \n#       -HELP -          #\n############################               \n                                           \nUse w, a, s d to move\n           \nType e to selct or q to go back\n                \nUse \"`\"\n to enter debug mode                   \nAnd most importantly be careful out there\n\npress \"q\" to go back ---->\n",
         "seed_prompt": "If you have a seed, enter it now, otherwise press enter:"
     },
     "player": {
         "health": 100,
         "stamina": 100,
         "inventory": []
+    },
+    "enemy_1": {
```

Comment:
As can be seen here the db.json file can get quite large between iterations with garbage data that will be overwritten per run so you need to add a ".gitignore" file to the repository with a single line "db.json" so it wont be committed to github for organization. Make sure to research .gitignore files if you don't understand how they work

Reply:
I believe the write function will automatically generate the db.json file from our init_db() function which is why it doesn't need to be in this repo.